cli/command/container/opts.go

```diff
@@ -899,3 +921,12 @@ func validateAttach(val string) (string, error) {
 	}
 	return val, errors.Errorf("valid streams are STDIN, STDOUT and STDERR")
 }
+
+func convertToNetworkmode(netmode opts.NetworkOpt) container.NetworkMode {
+	networkIDOrName := "default"
+	for _, net := range netmode.Value() {
+		networkIDOrName = net.Target
+		break
```

Comment:
yes I thought of that as well. But I saw that the other options like port were converting to lower case. So I decided to maintain the uniformity.